src/requirements/data/colleges/asPreFA2020.ts

```diff
@@ -0,0 +1,201 @@
+import { Course, CollegeOrMajorRequirement } from '../../types';
+import { courseIsFWS, includesWithSingleRequirement } from '../checkers-common';
+
+const casPreFA2020Requirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'A&S Credits',
+    description:
+      '100 credits in Arts & Sciences are required.' +
```

Comment:
Nit: I would add a space at the end so that it's  '100 credits in Arts & Sciences are required. '